todo_app/trelloclient.py

```diff
@@ -0,0 +1,106 @@
+import requests
+import os
+
+class Card:
+    def __init__(self, json, statusName):        
+        self.listid = json['idList']
+        self.id = json['id']                     
+        self.status = statusName
+        self.title = json['name']
+        self.due = json['due']
+        self.desc = json['desc']
+
+class TrelloClient:    
+    def __init__(self):
+        self.url = os.environ['TRELLO_BASE_URL'] 
+        self.key = os.environ['TRELLO_API_KEY']
+        self.token = os.environ['TRELLO_API_SECRET']
+        self.header = "application/json"
+
+        self.headers = {
+            "Accept": self.header
+        }
+
+        self.query = {
+            'key' : self.key,
+            'token': self.token
+        }
+
+class TrelloBoard(TrelloClient):
```

Comment:
These classes are slightly confusing named, a TrelloBoard actually represents a class that reads boards on Trellos, rather than representing the board itself - `TrelloBoardClient` etc. would be clearer naming

Reply:
Agree. thank you.  I have renamed most of my class so there are no confusion.